<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GUIDesktop</class>
 <widget class="QMainWindow" name="GUIDesktop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>370</width>
    <height>550</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>370</width>
    <height>550</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>370</width>
    <height>550</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string notr="true">PBX-Viewer: Desktop</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_Socket">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>20</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>SOCKET</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_PBX">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>120</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>PBX</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="linea_arriba">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>351</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_puerto">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>58</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Puerto: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_puerto">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>60</y>
      <width>113</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_tipoPBX">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>160</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo PBX: </string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboB_pbxs">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>160</y>
      <width>161</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_iniciar">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>450</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Iniciar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_parar">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>133</x>
      <y>450</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Parar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_salir">
    <property name="geometry">
     <rect>
      <x>253</x>
      <y>450</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Salir</string>
    </property>
   </widget>
   <widget class="Line" name="linea_abajo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>430</y>
      <width>341</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_nuevo">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>61</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Nuevo</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_editar">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>61</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Editar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton_eliminar">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>210</y>
      <width>81</width>
      <height>29</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Eliminar</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>351</width>
      <height>171</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>349</width>
       <height>169</height>
      </rect>
     </property>
     <widget class="QPlainTextEdit" name="texarea">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>351</width>
        <height>171</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>370</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArchivo">
    <property name="title">
     <string>Archivo</string>
    </property>
    <addaction name="menuIniciar"/>
    <addaction name="menuParar"/>
    <addaction name="menuSalir"/>
   </widget>
   <widget class="QMenu" name="menuEdici_n">
    <property name="title">
     <string>Edición</string>
    </property>
    <widget class="QMenu" name="menuIdioma_2">
     <property name="title">
      <string>Idioma</string>
     </property>
     <addaction name="actionEspanol"/>
     <addaction name="actionIngles"/>
     <addaction name="actionPortugues"/>
    </widget>
    <addaction name="menuIdioma_2"/>
   </widget>
   <widget class="QMenu" name="menuPBX">
    <property name="title">
     <string>PBX</string>
    </property>
    <addaction name="menuNuevoPerfil"/>
    <addaction name="menuModificarPerfil"/>
    <addaction name="menuEliminarPerfil"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAyuda"/>
    <addaction name="menuAcercade"/>
    <addaction name="menuAcercadeQt"/>
   </widget>
   <addaction name="menuArchivo"/>
   <addaction name="menuEdici_n"/>
   <addaction name="menuPBX"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="menuSalir">
   <property name="text">
    <string>Salir</string>
   </property>
  </action>
  <action name="menuNuevoPerfil">
   <property name="text">
    <string>Nuevo Perfil</string>
   </property>
  </action>
  <action name="menuModificarPerfil">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Modificar Perfil</string>
   </property>
  </action>
  <action name="menuEliminarPerfil">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Eliminar Perfil</string>
   </property>
   <property name="font">
    <font/>
   </property>
  </action>
  <action name="menuAcercade">
   <property name="text">
    <string>Acerca de</string>
   </property>
  </action>
  <action name="menuAcercadeQt">
   <property name="text">
    <string>Acerca de Qt</string>
   </property>
  </action>
  <action name="actionEspanol">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Español</string>
   </property>
  </action>
  <action name="actionIngles">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Inglés</string>
   </property>
  </action>
  <action name="actionPortugues">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Portugués</string>
   </property>
  </action>
  <action name="actionAyuda">
   <property name="text">
    <string>Ayuda</string>
   </property>
  </action>
  <action name="menuIniciar">
   <property name="text">
    <string>Iniciar</string>
   </property>
  </action>
  <action name="menuParar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Parar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>boton_salir</sender>
   <signal>clicked()</signal>
   <receiver>GUIDesktop</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>307</x>
     <y>470</y>
    </hint>
    <hint type="destinationlabel">
     <x>352</x>
     <y>487</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>menuSalir</sender>
   <signal>activated()</signal>
   <receiver>GUIDesktop</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
